dev/com.ibm.ws.logging_fat/fat/src/com/ibm/ws/logging/fat/ConsoleFormatTest.java

```diff
@@ -59,13 +59,16 @@
     private static final String DEV_FORMAT = "dev";
     private static final String SIMPLE_FORMAT = "simple";
     private static final String DEPRECATED_BASIC_FORMAT = "basic";
+    private static final String TBASIC_FORMAT = "tbasic";
     private static final String INVALID_CONSOLE_FORMAT = "simples";
 
     private static final String SERVER_NAME = "com.ibm.ws.logging.consoleformat";
     private static final String SERVER_ENV_NAME = "com.ibm.ws.logging.consoleformatenv";
 
     private static final String DEV_FORMAT_REGEX_PATTERN = "([A-Z]{3,}   )"; // Matches with line that has [<LOG_LEVEL>   ] in the beginning. e.g. [AUDIT   ]
     private static final String SIMPLE_FORMAT_REGEX_PATTERN = "(\\[\\d{1,4}.*)"; // Matches with line that [23/02/20 ... ] in the beginning.
+    private static final String TBASIC_FORMAT_REGEX_PATTERN = "([a-zA-Z0-9- ]{8} [aA-zZ]{1,13} [aA-zZ]{1}\\s{3})";
+    private static final String TBASIC_FORMAT_REGEX_PATTERN_CLASS = "([a-zA-Z0-9- ]{8} [aA-zZ]{1,13} [aA-zZ]{1}\\s{1})";
```

Comment:
This regex failed to match this valid line.
```
[17/03/21 13:42:33:328 EDT] 00000063 servlet       I com.ibm.ws.webcontainer.servlet.ServletWrapper init SRVE0242I: [badapp] [/badapp] [com.ibm.ws.lumberjack.badness.Angry]: Initialization successful.
```

Reply:
Resolved.